scripts/zh-cn.json

```diff
@@ -0,0 +1,81 @@
+  "2_0_merge_end-0-0": "打开这个文件： <cmd>accounts.csv</cmd>. 正如你所见，这个文件里有很多账号密码.一个黑客组织发现收集了这些信息.但是有个问题，其中只有一小部分的账号对我们有用，但我们也不知道是其中的哪些.我们需要找到三个银行家账户里的至少两个.这些服务器承载着安全密钥(<cmd>.key</cmd> 文件) ，在之后会对我们很有用.你搜集到这些文件以后再回到自己的服务器来.",
+  "2_0_merge_end-0-1": "打开一个CSV文件，使用指令 <cmd>cat filename.csv</cmd>.",
+  "2_1_key_end-0-0": "很好……你现在需要p̶̨̨͈̍͑̔͜r̵̬͉̅e̵̞̓p҉a҉r҉e҉",
+  "2_1_key_end-1-0": "我做到了！我暂时禁言了其他用户，我们俩需要搞快点……他随时都可能夺回掌控权……我知道你身上发生了什么，我会帮助你的.先装做跟随他的指令，我之后会来找你……",
+  "2_1_key_end-2-0": "… p̸̯͇̭͛̓̿̇͝r̸̞͂ė̶̻̤̹p̸a̷r̵e̵ 攻击银行.",
+  "2_1_key_end-3-0": "…",
+  "2_1_key_end-4-0": "哈，发生什么了？",
+  "2_1_key_end-4-1": "别妄想逞英雄——你只会后悔的.你唯一需要做的就是听好我的命令.",
+  "3_0_demoEnd-0-0": "[DEMO结束]",
+  "3_0_demoEnd-0-1": "你好！谢谢游玩我们的DEMO！如果你喜欢这款游戏，以及这款游戏的开发历史的话，能在itch.io或者Steam找到我们的完整版:) 祝你有美好的一天，以及我希望，等会还能再见到你！",
+  "3_0_attack_intro-0-0": "想完成这次攻击，我在等你获取你需要的所有可执行文件.进这个服务器去找找信息： <ip>{{ip}}</ip>，用户名是 <username>{{username}}</username>，密码是 <password>{{password}}</password>，把登陆以后所在文件夹的三个二进制文件都下载下来.",
+  "3_1_train_scanUsers-0-0": "干得还行，目前为止你还没让我失望.我希望你能继续这么有效率地完成我的要求.你现在需要学会怎么使用这些工具中的其中之一，也就是 <cmd>scanUsers</cmd>，它会检测目标服务器上能供你使用的用户账户.在攻击我们的目标之前，你试验一下这家伙能不能正确工作.你可记住了，我不想看到你犯任何错误.用之前你拼凑出来的那个服务器列表试试手吧.",
+  "3_1_train_scanUsers-1-0": "扫描系统用户，在禁用防火墙之后，使用指令 <cmd>scanUsers address</cmd>.",
+  "3_2_attack_intro_end-0-0": "你学会怎么用了？行，我们可以开始做正事了.",
+  "3_2_attack_intro_end-1-0": "记住，这事没有任何容错.我希望你的名声不是弄虚作假的.那我们开始干活！",
+  "4_0_diversion_intro-0-0": "要辅助我们的攻击，你需要黑进一些服务器来搭建僵尸进程去分流.如果你不精准地遵循我的指令的话，你就会被他们抓住.成功连接 <ip>{{ip}}</ip> 之后我就会跟你解释怎么创建僵尸进程.",
+  "4_0_diversion_intro-1-0": "搞快点！我不想等你磨蹭！",
+  "4_1_diversion_firstZombie-0-0": "你需要在这个服务器上搭建僵尸程序.这之后系统会启动一个后台进程，在我们发布后续的攻击命令之前都会保持空闲状态.现在可是个重要时刻，你可根本不要期望这个计划出什么岔子.",
+  "4_1_diversion_firstZombie-0-1": "搭建僵尸程序，使用指令 <cmd>zombieMaker</cmd>.",
+  "4_2_diversion_otherZombies-0-0": "重复之前的指令——连接服务器之后使用zombieMaker——来搞这台服务器： <ip>{{ip1}}</ip> 以及这台：<ip>{{ip2}}</ip>.",
+  "5_0_front_attack-0-0": "终于终于，最后一块拼图拼上了.等会我们只要激活这些僵尸，让它们去攻击客户端网站，银行的防火墙就会失效.这就是个完美的计划，你可根本别想着偏离计划.",
+  "5_0_front_attack-1-0": "在宕机时间里，我们就要攻击我们的目标了.",
+  "5_0_front_attack-2-0": "就是现在.准备开始僵尸攻击.瞄准 <ip>{{ip}}</ip>.",
+  "5_0_front_attack-3-0": "开始僵尸攻击，使用指令 <cmd>zombieAttack address</cmd>.",
+  "5_1_front_check-0-0": "停停停.先别做任何事.我需要检查分流是不是正常：",
+  "5_1_front_check-1-0": "好了！开始攻击了，连接服务器 <ip>{{ip}}</ip>. 然后赶快！",
+  "6_0_dmz_intern_attack-0-0": "我们现在在DMZ（隔离区）服务器上.我们现在能穿行在他们的网络架构里，一台一台接近目标数据库服务器.我们没争取到多少时间，所以你得赶快.",
+  "6_0_dmz_intern_attack-1-0": "准备好了？好！先去服务器 <ip>{{ip}}</ip>，这台服务器部署着隔离区保护内网的防火墙.",
+  "6_1_firewall_process-0-0": "好，现在，杀掉进程 <cmd>firewall</cmd>，然后你就能连接服务器地址 <ip>{{ip}}</ip>.",
+  "6_1_firewall_process-1-0": "终止一个进程，使用指令 <cmd>kill processName</cmd>.",
+  "6_2_web_intern-0-0": "你现在在一台内部网页服务器上，还剩一步要做：够到内部数据库服务器，地址是 <ip>{{ip}}</ip>.",
+  "6_3_security_alert-0-0": "已检测到未授权连接！正在采取紧急情况措施.",
+  "6_3_security_alert-disconnected": "连接已超时.",
+  "6_3_security_alert-1-0": "哦不……我来得太晚了……她还是让你攻击了银行.抱歉，我不得不告诉你……你成功不了的.",
```

Comment:
Here, translation tools tell me the last sentence is usng "can't succeed" as a verb but it should be "couldn't succeed". Meaning, even if you wanted and tried hard, it was impossible.

Reply:
I think for this point, in Chinese "can't" and "couldn't" are basically interchangeable - the subtle difference is introduced by the context around the word. Previously "I have to tell you" already brings "(by all means) you cannot succeed (at all)" from my point of view.

Reply:
Okay I see ! Then I close this conversation. Thanks :)